app/Models/Hit.php

```diff
@@ -22,9 +22,19 @@ class Hit extends Model
     public static function forTimePeriod(string $period = 'week'): array
     {
         $now = CarbonImmutable::now();
+        $currentPeriodStart = $now->sub($period, 1)->addSecond();
```

Comment:
So we're being very careful around the second (which is good!), but sometimes cron jobs don't start at *exactly* the same moment. I'm wondering... should we, instead of assuming everything is based around "now", instead allow someone to pass a time in here as a second parameter?

Reply:
If we were to do that I assume we would hard code it in the cron code that calls this (e.g. "Sunday this week at 8am" or something)